app/Main.hs

```diff
@@ -12,4 +12,5 @@ main = do
     reportFile <- maybe (print "You must provide a report .yaml file" *> exitFailure)
                         pure $
                         head args
```

Comment:
Is this still valid? Feels like there needs to be a more complicated handling of args now.

Reply:
It should still be fine. It’s either `stack exec statsbot file.yaml` or `stack exec statsbot file.yaml --test`.

Reply:
I’ll merge it, but let me know if you think I should change anything or add comments

Reply:
But `statsbot --test file.yaml` would produce a confusing error

Reply:
That’s true, but the only thing that mentions `--test` is a comment in `daily_product.yaml` that can be copy/pasted. I figured this would be better than a comment that suggested editing the statsbot project to test.

But would it be enough to just add some basic validation that the first arg ends in `.yaml` and is a file?